sql.txt

```diff
@@ -3,9 +3,16 @@ SELECT with JOIN practice:
 Join the OrderDetails and Products tables, to show a report where the columns are OrderID, ProductName, and Quantity.
 Paste the SQL statement you used below.
 
+SELECT OrderID, ProductName, Quantity FROM OrderDetails
+	INNER JOIN Products ON OrderDetails.ProductID = Products.ProductID;
 
-Join the Orders, OrderDetails, and Employees tables to return a report with with the EmployeeName, ProductID, and Quantity.
-Paste the SQL statement you used below.  Hint: EmployeeName is not a column in the Employee table, but you can generate a
-report with EmployeeName by starting your SQL this way: SELECT (Employees.FirstName || " " || Employees.LastName) AS EmployeeName,
 
+Join the Orders, OrderDetails, and Employees tables to return a report with the EmployeeName, ProductID, and Quantity.
+Paste the SQL statement you used below.  
+
+Hint: EmployeeName is not a column in the Employee table, but you can generate a
+report with EmployeeName by starting your SQL this way: SELECT (Employees.FirstName || " " || Employees.LastName) AS EmployeeName,
 
+SELECT (Employees.Firstname || " " || Employees.LastName) AS EmployeeName, ProductID, Quantity FROM Orders
+    INNER JOIN OrderDetails ON Orders.OrderID = OrderDetails.OrderID
+    INNER JOIN Employees ON Orders.EmployeeID = Employees.EmployeeID
```

Comment:
This is almost correct. For ProductID and Quantity you're going to need to put the refer to the tables for these columns like you did with Employees.Firstname. For example, OrderDetails.Quantity.